src/index.js

```diff
@@ -143,6 +144,34 @@ export default function ({ template, types }) {
             }
           },
         });
+
+        programPath.scope.crawl();
```

Comment:
Yes this crawl is needed as it actually computes what is removed in the first traversal.
If we remove this, in the second traversal, we will get the same AST as the first pass, and it will still have referencePaths to
```
ABC.propTypes = {
  name: PropTypes.string // Reference Path to this.
}
```

What I am doing is a two pass algorithm. I could not figure out how to do it in one pass.